src/css/page_sections/Menu/_SubmenuItem.scss

```diff
@@ -0,0 +1,34 @@
+.coa-SubmenuItem {
+  list-style: none;
+  &:hover {
+    background: $coa-color-another-grayish-purple;
```

Comment:
same here -- checkout the color set I defined in my branch. Celine has a good amount of organization in her stickersheet to keep our styles from getting unruly that I think we should stick to and have clearly named.